src/continuation/continuation_clustering.jl

```diff
@@ -0,0 +1,170 @@
+  [`basins_fractions_continuation`](@ref). This method computes the Features accross a 
+  range of parameters before performing a clustering into classes using a special metric, 
+  see [^Gelbrecht2020]. 
+
+The function takes as an input a `mapper` that maps initial conditions to attractors 
+  using the featurizing method [^Stender2021]. See [`AttractorMapper`](@ref) for how to  
+  use the `mapper`.
+
+## Keyword Arguments
+- `prange` Range of parameter to analyze.
+- `pidx` Number or symbol of the parameter to change in the array of parameters of the dynamical system. 
+- `ics` Sampler function to generate initial conditions to sample. 
+- `samples_per_parameter = 100` Number of samples per parameter.
+- `par_weight = 1` The distance matrix between features has a special extra  weight that 
+  is proportional to the distance |pi - pj| between the parameters of each features. This 
+  keyword argument is the weight coeficient that ponderates the distance matrix.
+- `ϵ_optimal = 1.` This is the optimal radius for the DBSCAN clustering algorithm. It should be set 
+  manually. 
+- `mmap_limit = 20000` this parameter sets the limit of features that should be clustered using only the 
+  RAM memory. Above this limit the program uses a memory map based array to store the distance 
+  matrix on the disk. 
+
+## Description
+
+The method first simulate and compute a set of statistics on the trajectories, for example the mean 
+ and standard deviation of the trajectory data points. Once all the featured statistics have been 
+ computed, the algorithm computes the distance between each feature. A special weitgh term is added 
+ so that the distance between features with different parameters is increased. It helps to discriminate
+ between attractors with very different parameters. At last, the distance matrix is clustered with the 
+ DBSCAN algorithm
+
+[^Gelbrecht2020]:
+    Gelbrecht, M., Kurths, J., & Hellmann, F. (2020). Monte Carlo basin bifurcation
+    analysis. New Journal of Physics, 22(3), 033032.
+
+[^Stender2021]:
+    Stender & Hoffmann, [bSTAB: an open-source software for computing the basin
+    stability of multi-stable dynamical systems](https://doi.org/10.1007/s11071-021-06786-5)
+"""
+function ClusteringAcrossParametersContinuation(
+        mapper::AttractorsViaFeaturizing;
+        info_extraction = mean_across_features
+        # TODO: Here we can add more keywords regarding how to cluster across parameters.
+    )
+    return (; mapper, info_extraction)
+end
+
+function mean_across_features(fs)
+    means = zeros(length(first(fs)))
+    N = length(fs)
+    for f in fs
+        for i in eachindex(f)
+            means[i] += f[i]
+        end
+    end
+    return means ./ N
+end
+
+
+
+function basins_fractions_continuation(
+        continuation::NamedTuple, prange, pidx, ics::Function;
+        samples_per_parameter = 100, show_progress = true, par_weight = 1, 
+        ϵ_optimal = 1., mmap_limit = 20000
+    )
```

Comment:
All the keywords besides `samples_per_parmeter, show_progress` should instead be given s arguments to the struct `ClusteringAcrossParametersContinuation`. Then, use that struct in the `basins_fractions_continuation` method.